src/plugins/intel_gpu/src/graph/graph_optimizer/oooq_memory_dependencies.cpp

```diff
@@ -134,13 +137,15 @@ void oooq_memory_dependencies::run(program& p) {
     };
 
     unsigned int A = 0;
-    auto itr_A = processing_order.begin();
+    auto itr_A = processing_order_except_const.begin();
 
-    while (itr_A != processing_order.end()) {
+    while (itr_A != processing_order_except_const.end()) {
         if (suspect_nodes.is_set(A)) {
             std::vector<std::pair<program_node*, unsigned int>> deps;
             for (const auto& dep : (*itr_A)->get_dependencies()) {
-                deps.emplace_back(dep, user_map.at(dep));
+                if (!dep->is_type<data>()) {
```

Comment:
Is it necessary? It looks like that all items in `processing_order_except_const` already are not `data` types.

Reply:
I'm sorry for my misunderstanding.